mcs/build/tests.make

```diff
@@ -262,7 +262,9 @@ ifdef HAVE_CS_XTESTS
 XTEST_HARNESS_PATH = $(topdir)/../external/xunit-binaries
 XTEST_HARNESS = $(XTEST_HARNESS_PATH)/xunit.console.exe
 XTEST_HARNESS_FLAGS := -noappdomain -noshadow -parallel none -nunit TestResult-$(PROFILE)-xunit.xml
-XTEST_TRAIT := -notrait category=failing -notrait category=nonmonotests -notrait Benchmark=true -notrait category=outerloop
+XTEST_TRAIT := -notrait category=failing -notrait category=nonmonotests -notrait category=nonnetfxtests -notrait Benchmark=true -notrait category=outerloop
```

Comment:
We cannot use `nonnetfxtests` that would disable tests we expect to pass. There should be only few cases where we need to disable such tests and should use Mono test moniker